SignalServiceKit/src/Contacts/TSThread.h

```diff
@@ -188,11 +189,11 @@ typedef NS_ENUM(NSInteger, TSThreadFriendRequestStatus) {
 
 #pragma mark - Loki Friend Request Handling
 
-- (TSThreadFriendRequestStatus)getFriendRequestStatus;
+- (void)setFriendRequestStatus:(TSThreadFriendRequestStatus)friendRequestStatus withTransaction:(YapDatabaseReadWriteTransaction *_Nullable)transaction;
```

Comment:
It might be better to have a function declaration without the transaction, rather than the user explicitly setting the transaction to `nil`.
`- (void)setFriendRequestStatus:(TSThreadFriendRequestStatus)friendRequestStatus`
which passes `nil` into this function.

Reply:
That'd override the setter of the property

Reply:
would it make more sense to name it to `saveFriendRequestStatus` instead?